src/post-merge/composer_install.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+
+composer -v > /dev/null 2>&1
```

Comment:
Is better to check that command exists

```suggestion
command -v composer > /dev/null 2>&1
```